app/models/user.rb

```
@@ -4,6 +4,8 @@
 class User < ApplicationRecord
   has_secure_password
   has_many :sessions, dependent: :destroy
+  has_many :rsvps
```

RuboCop (1.75.5)

[convention] Specify a `:dependent` option.

(Rails/HasManyOrHasOneDependent)